docs/Release_notes.md

```diff
@@ -1,6 +1,66 @@
 CARMA Platform Release Notes
 ----------------------------
 
+Version 3.6.0, released June 29th, 2021
+-------------------------------------
+
+**Summary**
+Carma-platform release version 3.6.0 is comprised of five major enhancements. First, Added ADS Lane Change, unobstructed lane change. Second CTM Move-over law –When receiving a request from an emergency vehicle, CARMA Platform plans to safely pull over on a shoulder lane. Third, Added Geofence speed, Gap control and lane closure. And fourth, added Cloud world model, lane closure. Along with the above enhancements, several bug fixes and CI related enhancements are included in this release.
```

Comment:
I'm concerned about this phrasing "CARMA Platform plans to safely pull over on a shoulder lane."
We have not demonstrated any functionality where the vehicle pulls over. When we get the request from an emergency vehicle the vehicle will change lanes to avoid it. Additionally, I have issues with the phrasing "safely" which in my opinion implies object avoidance behavior which was not enabled in this release. 

Other notes
* Gap control was included but not tested. I'm not sure we should point it out as a supported feature in this release
* "ADS Lane change and unobstructed lane change" are redundant and could probably just be listed as "ADS unobstructed lane change". 
* "Cloud world model" is a very unclear term. I would say "added integration with carma-cloud"
@snallamothu Looks like you sent these same ones to the client for review. Please advise on changes

Reply:
"CARMA Platform plans to safely pull over on a shoulder lane" should be changed to "CARMA Platform plans move over to the adjacent open lane".
 Gap control - leave it in because that is one of the attributes we checked for presence to meet the requirements though we didn't test with two vehicles.
I am OK with "ADS unobstructed lane change" but aren't we using CLC.  If that is correct, we should change ADS to CLC.
I am OK with changing the "Cloud world model" to "carma-cloud integration".

Reply:
Per our discussion on slack we will use "ADS unobstructed lane change" as we have not tested with another vehicle

@SaikrishnaBairamoni Please make the changes Sudhakar has listed